rialto_airflow/harvest/crossref.py

```diff
@@ -67,7 +67,8 @@ def get_dois(dois: Iterable[str], tries=5) -> Iterable[dict]:
 
             # According to Crossref API error messages the DOI must be of the form:
             # doi:10.prefix/suffix where prefix is 4 or more digits and suffix is a string
-            if m := re.match(r"^doi:10\.(\d+)/.+$", doi):
+            # the string must not contain whitespace
+            if m := re.match(r"^doi:10\.(\d+)/\S+$", doi):
```

Comment:
is this just an additional safeguard, because in theory, the cleanup method below should prevent whitespace from ever being sent?

Reply:
Yeah, that's true. It was here so I could move an existing DAG run along last week. It doesn't seem like it will do any harm to have it?

Reply:
yup, no harm, just making sure i understood correctly!